test/trieSetTest.mo

```diff
@@ -0,0 +1,28 @@
+import Nat "mo:base/Nat";
+import TrieSet "mo:base/TrieSet";
+import Word32 "mo:base/Word32";
+import Suite "mo:matchers/Suite";
+import M "mo:matchers/Matchers";
+import T "mo:matchers/Testable";
+
+let set1 = TrieSet.fromArray<Nat>([ 1, 2, 3 ], Word32.fromNat, Nat.equal);
+
+let suite = Suite.suite("TrieSet fromArray", [
```

Comment:
Awesome that you picked on `matchers` and added tests using it ❤️

Reply:
Yeah, this is very handy!